showcase/src/app/demo/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,22 @@
+"use strict";
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+const componentName: string = "StarkBreadcrumb";
+
+/**
+ * AAH Breadcrumb component page
```

Comment:
AAH was the old demo project. Could you please adapt this for "Demo Breadcrumb component page" ?